games-arcade/hikounomizu/additional-files/hikounomizu-0.9.2.rdef

```diff
@@ -0,0 +1,170 @@
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+	major = 0,
+	middle = 9,
+	minor = 2,
```

Comment:
Normally the recipes use a ".rdef.in" file and then generate the ".rdef" inside the INSTALL() section so that it doesn't need to be edited if there is a new release.  For example, the 7kaa [recipe](https://github.com/haikuports/haikuports/blob/master/games-strategy/7kaa/7kaa-2.15.4p1.recipe#L62) and [rdef.in](https://github.com/haikuports/haikuports/blob/master/games-strategy/7kaa/additional-files/7kaa.rdef.in) file

Reply:
Done! Thanks a lot for your feedback